.github/workflows/workflow.yml

```diff
@@ -42,6 +42,7 @@ jobs:
           dub test --build=unittest-cov
 
       - name: Upload coverage
+        continue-on-error: true
         uses: codecov/codecov-action@v1
```

Comment:
```suggestion
        uses: codecov/codecov-action@v1.0.5
```

Reply:
You can try specifying this version. Doing some research it seems this a more stable version and it fixes this issue for the majority.